### DISCUSSION TITLE
How to hide recursive?

### DISCUSSION THREAD

**Discussion by princyok** [TylerYep/torchinfo#249] [Q&A] [answered]
How do I hide recursive from repeating or showing. 

I tried
```python
torchinfo.summary(
    model, 
    input_data=data,
    row_settings={"no_recursive"},
    verbose=1
)
``` 
But that failed with ``ValueError: 'no_recursive' is not a valid RowSettings``. I was trying out the feature that was added by this [PR](https://github.com/TylerYep/torchinfo/pull/174). I tried this after updating torchinfo to the latest version using conda.

**Comment by TylerYep** [comment, marked as answer]
The setting is called `"hide_recursive_layers"`. I recommend importing `RowSettings` and using `RowSettings.HIDE_RECURSIVE_LAYERS` to be more clear. Hope this helps!
